sdk/cosmosdb/Azure.ResourceManager.CosmosDB/src/autorest.md

```diff
@@ -108,6 +108,7 @@ rename-rules:
   Etag: ETag|etag
   Mongodb: MongoDB
   VNet: Vnet
+  API: Api
 
 override-operation-name:
   RestorableMongodbDatabases_List: GetRestorableMongoDBDatabases
```

Comment:
I do not see the new `GetRestorableMongoDBResources` method which returns `Pageable<RestorableMongoDBResourceData>`. I think it's because its method signature is the same as the old method  which you added back in the custom code (only difference is the return type which is not part of the method signature). The generator will stop generating the method if it has found the same method signature in custom code.

Therefore, we need to update the method name to have a different method signature. Please update `RestorableMongodbResources_List` in line 116 with a new name as the old method already took `GetRestorableMongoDBResources`, I suggest `GetAllRestorableMongoDBResourceData`. Also add a similar config for `RestorableSqlResources_List: GetAllRestorableSqlResourceData`